.github/ISSUE_TEMPLATE/bug.yml

```
@@ -0,0 +1,58 @@
+name: 🐞 Bug Report
+description: Tell us about something that's not working the way we (probably) intend.
+labels: ["Kind:Bug", "State:Triage"]
+body:
+  - type: input
+    id: elixir-version
+    attributes:
+      label: Elixir version
+      description: Use `elixir -v` to find the Elixir version.
+    validations:
+      required: true
+
+  - type: input
+    id: db-version
+    attributes:
+      label: Database and Version
+      description: >
+        The database and its version (PostgreSQL 9.4, MongoDB 3.2, etc.)
+    validations:
+      required: true
+
+  - type: textarea
+    id: ecto-version
+    attributes:
+      label: Ecto Versions
+      description: Use `mix deps` to find the dependency versions.
+    validations:
+      required: true
+
+  - type: textarea
+    id: db-adapter-version
+    attributes:
+      label: Database Adapter and Versions
```

```suggestion
      label: Database Adapter and Versions (postgrex, myxql, etc) 
```

Or maybe a placeholder is better?